cmd/create.go

```diff
@@ -29,7 +29,7 @@ var componentCreateCmd = &cobra.Command{
 	Short: "Create a new component",
 	Long: `Create a new component to deploy on OpenShift.
 
-If component name is not provided, component type value will be used for the name.
+If component name is not provided, it'll be auto-generated.
```

Comment:
Argh. Sorry this wasn't updated before. But correct would be `If a component name is not provided`

Reply:
Done